include.js

```diff
@@ -114,11 +114,11 @@ let cards = [
     githubLink: 'https://github.com/MaKloudz'
   },
   {
-    artName: 'That Animation',
-    pageLink: './Art/MaKloudz/index.html',
-    imageLink: './Art/MaKloudz/dat-animation.gif',
-    author: 'Blessing Mutava',
-    githubLink: 'https://github.com/MaKloudz'
```

Comment:
It looks like you are deleting the entry from another user? Can you please look into that?